peregrine/tracking.py

```diff
@@ -721,6 +721,7 @@ def __init__(self, params):
     params['code_freq_init'] = params['acq'].doppler * \
         gps_constants.l2c_chip_rate / gps_constants.l2
     params['chipping_rate'] = gps_constants.l2c_chip_rate
+    params['sample_index'] = 0
```

Comment:
How to support the use case, when only L2C tracking is enabled, and offset is defined?